tkdesigner/elements.py

```diff
@@ -0,0 +1,172 @@
+from tkdesigner.constants import ASSETS_PATH
+from jinja2 import Template
+from tkdesigner.template import TEMPLATE
+
+
+TEXT_INPUT_ELEMENT_TYPES = {
+    "TextArea": "Text",
+    "TextBox": "Entry"
+}
+
+
+class FigmaFrame:
+    """Represents the Figma Frame for which the GUI will be generated.
+    
+    Attributes:
+        width: Width of the window.
+        height: Height of the window.
+        elements: Child GUI elements.
+        bg_color: Color of the GUI background.
+    """
+    def __init__(self, width, height, bg_color="#FFFFFF", elements=[]):
+        self.width = width
+        self.height = height
+        self.elements = elements
+        self.bg_color = bg_color
+
+
+    def to_code(self, template=TEMPLATE):
+        t = Template(template)
+        return t.render(window=self, elements=self.elements, assets_path=ASSETS_PATH)
+
+
+class FigmaElement:
+    """Represents each Figma element in the UI.
+
+    Each element has a position, size, and a unique ID.
+
+    Attributes:
+        id_: ID unique to that element, useful for variable names.
+        x: X position coordinate of the element.
+        y: Y position coordinate of the element.
+        width: Width of the element.
+        height: Height of the element.
+    """
+
+    def __init__(self, id_, x, y, width, height):
+        self.id_ = id_
+        self.x = x
+        self.y = y
+        self.width = width
+        self.height = height
+
+    def to_code(self):
+        raise NotImplementedError("to_code() not implemented on element class.")
+
+
+class RectangleElement(FigmaElement):
+    def __init__(self, id_, x, y, width, height, fill_color):
+        super().__init__(id_, x, y, width, height)
+
+        self.fill_color = fill_color
+
+    def to_code(self):
+        return f"""
+canvas.create_rectangle(
+    {self.x},
+    {self.y},
+    {self.x + self.width},
+    {self.y + self.height},
+    fill="{self.fill_color}",
+    outline="")
+"""
+
+
+class ButtonElement(FigmaElement):
+    def __init__(self, id_, x, y, width, height, image_path):
+        super().__init__(id_, x, y, width, height)
+
+        self.image_path = image_path
+
+    def to_code(self):
+        return f"""
+button_image_{self.id_} = PhotoImage(file=rel_asset_path("{self.image_path}"))
+button_{self.id_} = Button(
+    image=button_image_{self.id_},
+    borderwidth=0,
+    highlightthickness=0,
+    command=lambda _: print("button_{self.id_} clicked"),
```

Comment:
lambda is requiring 1 argument which is not being passed and also not needed. It's throwing error. 

Fix -> 
`command=lambda: print("button_{self.id_} clicked")`

Reply:
Fixed in https://github.com/ParthJadhav/Tkinter-Designer/pull/67/commits/f10414b845e1fb65eac7ee714ea97696bb720124